libs/novui/src/components/modal/Modal.tsx

```diff
@@ -0,0 +1,26 @@
+import { Modal as ExternalModal, ModalProps as ExternalModalProps } from '@mantine/core';
+import { CoreProps } from '../../types';
+import { modal } from '../../../styled-system/recipes';
+import { splitCssProps } from '../../../styled-system/jsx';
+import { css, cx } from '../../../styled-system/css';
+
+interface IModalProps extends ExternalModalProps, CoreProps {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** ‏Please be careful about directly extending Mantine props, and instead use `Pick<>` to select specific ones. Otherwise we couple ourselves and expose too many options that allow for deviating from the design system